featuretools/primitives/standard/transform/time_series/yeo_johnson.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+from scipy import stats
+from woodwork.column_schema import ColumnSchema
+
+from featuretools.primitives.base import TransformPrimitive
+
+
+class YeoJohnson(TransformPrimitive):
+    """Applies a transformation to help your data resemble a normal distribution,
+        commonly known as the Yeo-Johnson power transformation. Works with negative,
+        positive, and zero values. Often used with time series data, to reduce the variance.
+
+    Args:
+        lmbda (int, float, optional): The exponent to use for the transformation.
+            Default is None.
+
+    Examples:
+        >>> yeo_johnson = YeoJohnson()
+        >>> transformed = yeo_johnson([1, -10, 5, -4, 2])
+        >>> transformed = [round(x, 2) for x in transformed.tolist()]
+        >>> transformed
+        [1.12, -6.26, 7.1, -2.99, 2.43]
+
+        You can specify the lambda to use for the transformation
+
+        >>> yeo_johnson = YeoJohnson(lmbda=-1)
+        >>> transformed = yeo_johnson([1, -10, 5, -4, 2])
+        >>> transformed = [round(x, 2) for x in transformed.tolist()]
+        >>> transformed
+        [0.5, -443.33, 0.83, -41.33, 0.67]
+
+    """
+
+    name = "yeo_johnson"
+    input_types = [
+        ColumnSchema(semantic_tags={"numeric"}),
+    ]
+    return_type = ColumnSchema(semantic_tags={"numeric"})
```

Comment:
Same here, I think the return type can be Double logical type

Reply:
Fixed
https://github.com/alteryx/featuretools/pull/2387/commits/475efc8709837593cbea0a7194cb90453f25dd6b